modules/engage-theodul-core/src/main/resources/ui/js/engage/core.js

```diff
@@ -459,6 +459,23 @@ define(['require', 'jquery', 'underscore', 'backbone', 'mousetrap', 'bowser', 'b
   }
 
   function loadTemplate(plugin, plugin_name, plugin_path) {
+
+    // The following code block contains the logic for the download tab in the theodul player.
```

Comment:
Instead of checking this here, could we check it in `modules/engage-theodul-plugin-tab-downloads/src/main/resources/static/main.js` and have the plugin potentially do nothing instead? That way, we have no plug-in specific code in here.